Lib/encodings/__init__.py

```diff
@@ -69,6 +69,7 @@ def normalize_encoding(encoding):
 
 def search_function(encoding):
 
+    _aliases = aliases.aliases
```

Comment:
Don't understand the problem.
This sentence should be placed at below so that it does not affect the performance of the cache.

Reply:
> Don't understand the problem.

It will affect the encodings module's refcount in C level and reduce the refleaks.

> This statement should be placed at below so that it does not affect the performance of the cache.

Thanks for your comment. MAYBE removing this line and using aliases.aliases to replace `_aliases` is fine too :)

Reply:
Looking this comment: https://github.com/python/cpython/pull/21896#issuecomment-674474975

Reply:
> Looking this comment: [#21896 (comment)](https://github.com/python/cpython/pull/21896#issuecomment-674474975)

The usage of 'aliases.aliasesis' is very normal, maybe the root of the problem is not here.